lerna-debug.log

```diff
@@ -0,0 +1,11 @@
+0 silly argv {
```

Comment:
We forgot to add a *.log line in the .gitignore file, please do so we can avoid this files